devices/src/acpi.rs

```diff
@@ -254,3 +262,204 @@ impl BusDevice for AcpiPmTimerDevice {
         data.copy_from_slice(&counter.to_le_bytes());
     }
 }
+
+pub const COMMAND_ALLOCATE: u32 = 0x1;
+pub const COMMAND_ADD_POINTER: u32 = 0x2;
+pub const COMMAND_ADD_CHECKSUM: u32 = 0x3;
+
+pub const ALLOC_ZONE_HIGH: u8 = 0x1;
+pub const ALLOC_ZONE_FSEG: u8 = 0x2;
+
+pub const FW_CFG_FILENAME_TABLE_LOADER: &str = "etc/table-loader";
+pub const FW_CFG_FILENAME_RSDP: &str = "acpi/rsdp";
+pub const FW_CFG_FILENAME_ACPI_TABLES: &str = "acpi/tables";
+
+pub const SIGNATURE: [u8; 4] = *b"XSDT";
+pub const COMPILER_ID: [u8; 4] = *b"RVAT";
+
+#[cfg(all(feature = "fw_cfg", not(target_arch = "riscv64")))]
```

Comment:
there are so many `#[cfg(all(feature = "fw_cfg", not(target_arch = "riscv64")))]`, it might be worth it to have a dedicated module with a single feature gate?